XYZDrivers/src/main/java/com/xyzdrivers/controllers/AdminController.java

```diff
@@ -53,7 +56,13 @@ protected void processRequest(HttpServletRequest request, HttpServletResponse re
     @Override
     protected void doGet(HttpServletRequest request, HttpServletResponse response)
             throws ServletException, IOException {
-        processRequest(request, response);
+        try {
+            processRequest(request, response);
+        } catch (SQLException ex) {
+            Logger.getLogger(AdminController.class.getName()).log(Level.SEVERE, null, ex);
+        } catch (ClassNotFoundException ex) {
```

Comment:
Can these be combined into one catch? I think there's an A | B ex syntax.

Reply:
Yeah I saw Alex use that syntax, im not sure if I like it as you can't then perform logic based on the exception. But seeing as I only log it here its probs best to use the OR so il update. Cheers